docs/content/en/docs/reference/clusterspec/nutanix.md

```diff
@@ -0,0 +1,322 @@
+ 
+>**_NOTE:_** The taints provided will be used instead of the default control plane taint `node-role.kubernetes.io/master`.
+Any pods that you run on the control plane nodes must tolerate the taints you provide in the control plane configuration.
+> 
+
+### controlPlaneConfiguration.labels
+A list of labels to apply to the control plane nodes of the cluster. This is in addition to the labels that EKS Anywhere will add by default.
+ 
+Modifying the labels associated with the control plane configuration will cause new nodes to be rolled out, replacing the existing nodes.
+
+### workerNodeGroupConfigurations (required)
+This takes in a list of node groups that you can define for your workers. You may define one or more worker node groups.
+
+### workerNodeGroupConfigurations.count
+Number of worker nodes. Optional if `autoscalingConfiguration` is used, in which case count will default to `autoscalingConfiguration.minCount`.
+
+### workerNodeGroupConfigurations.machineGroupRef (required)
+Refers to the Kubernetes object with Nutanix specific configuration for your nodes. See `NutanixMachineConfig` fields below.
+
+### workerNodeGroupConfigurations.name (required)
+Name of the worker node group (default: `md-0`)
+
+### workerNodeGroupConfigurations.autoscalingConfiguration.minCount
+Minimum number of nodes for this node group’s autoscaling configuration.
+
+### workerNodeGroupConfigurations.autoscalingConfiguration.maxCount
+Maximum number of nodes for this node group’s autoscaling configuration.
+
+### workerNodeGroupConfigurations.taints
+A list of taints to apply to the nodes in the worker node group.
+ 
+Modifying the taints associated with a worker node group configuration will cause new nodes to be rolled-out, replacing the existing nodes associated with the configuration.
+ 
+At least one node group must not have `NoSchedule` or `NoExecute` taints applied to it.
+
+### workerNodeGroupConfigurations.labels
+A list of labels to apply to the nodes in the worker node group. This is in addition to the labels that EKS Anywhere will add by default.
+ 
+Modifying the labels associated with a worker node group configuration will cause new nodes to be rolled out, replacing the existing nodes associated with the configuration.
+
+### datacenterRef
+Refers to the Kubernetes object with Nutanix environment specific configuration. See `NutanixDatacenterConfig` fields below.
+
+### kubernetesVersion (required)
+The Kubernetes version you want to use for your cluster. Supported values: `1.23`, `1.22`, `1.21`, `1.20`
+
+## NutanixDatacenterConfig Fields
+
+### endpoint (required)
+The Prism Central server fully qualified domain name or IP address. If the server IP is used, the PC SSL certificate must have an IP SAN configured.
+
+### port (required) 
+The Prism Central server port. (Default: `9443`) 
+
+### insecure (optional)
+Set insecure to `true` if the Prism Central server does not have a valid certificate. (Default: `false`)
```

Comment:
I would add a note saying this is not really recommended for production deployment.
```suggestion
Set insecure to `true` if the Prism Central server does not have a valid certificate. This is not recommended for production use cases. (Default: `false`)
```